examples/build.gradle

```diff
@@ -0,0 +1,83 @@
+import org.apache.tools.ant.taskdefs.condition.Os
+
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+apply plugin: 'java'
+
+description = 'Ballerina - Persist Examples'
+
+def ballerinaDist = "${project.rootDir}/target/ballerina-runtime"
+def examples = ["redcross"]
```

Comment:
```suggestion
def examples = ["medical_center"]
```

Reply:
Fixed in [0b1c605](https://github.com/ballerina-platform/module-ballerina-persist/pull/5/commits/0b1c6053251a714640f3a3dd7f0879915715dbc3)